app/javascript/src/components/Dashboard/Notes/NoteTable.jsx

```diff
@@ -26,15 +27,20 @@ export default function NoteTable({
                 }}
               />
             </th>
-            <th className="text-left">Title</th>
-            <th className="text-left">Description</th>
+            <th className="text-left text-gray-500">Title</th>
+            <th className="text-left text-gray-500"> Description</th>
+            <th className="text-center text-gray-500">Tags</th>
+            <th className="text-center text-gray-500">Created Date</th>
+            <th className="text-center text-gray-500">Due Date</th>
+            <th className="text-center text-gray-500">Contact</th>
+            <th></th>
           </tr>
         </thead>
         <tbody>
           {notes.map(note => (
             <tr
               key={note.id}
-              className={"cursor-pointer bg-white hover:bg-gray-50"}
+              className={"group cursor-pointer bg-white hover:bg-gray-50"}
```

Comment:
Please remove this and add `nui-table--hover nui-table--actions` classname to the table.

The former adds hover effects to the table row and the latter adds hover effects to the `IconButtons`.